src/OpenIddict.Mvc/OpenIddictController.cs

```diff
@@ -38,7 +38,7 @@ public class OpenIddictController<TUser, TApplication> : Controller where TUser
         protected virtual OpenIddictOptions Options { get; }
 
         [HttpGet, HttpPost]
-        public async Task<IActionResult> Authorize() {
+        public virtual async Task<IActionResult> Authorize() {
```

Comment:
I don't have strong opinion, I expect that using custom views (which this PR addresses) is the predominant scenario, making controller action virtual seems like a dupe, having said that I'm not totally against this change.